server/src/main/java/com/hedera/block/server/consumer/LiveStreamObserverImpl.java

```diff
@@ -111,7 +111,7 @@ public void onNext(final BlockStreamServiceGrpcProto.BlockResponse blockResponse
      */
     @Override
     public void onError(final Throwable t) {
-        LOGGER.log(System.Logger.Level.ERROR, t);
+        LOGGER.log(System.Logger.Level.ERROR, "Unexpected consumer stream communication failure: {0}", t.getMessage());
```

Comment:
This can still just be `t` for the parameter, Throwable.toString() produces a good output, and passing the throwable will log the stack trace.

Reply:
<img width="1215" alt="image" src="https://github.com/hashgraph/hedera-block-node/assets/117216804/49976a0e-fcda-4de5-b440-bb540f2731e5">
* When I use just `t` it does output the stacktrace but it doesn't replace "{0}" (screenshot shows the unit test, the code and the error output below).  Is there a way to get it to replace "{0}" as well?

Reply:
Ahh, yes.  The flaws of the "string" assumption in JUL.  The throwable overload doesn't fill in parameters for the default formatters (because the formatters assume the messages are intended for localization strings, not actually for filling in data).
My apologies, I had forgotten about that limitation.
The workaround is to use `.formatted` as follows.
```suggestion
        LOGGER.log(System.Logger.Level.ERROR, "Unexpected consumer stream communication failure: %s".formatted(t), t);
```

Reply:
That works as expected.  We'll use that workaround going forward.  Thanks for the fix!